File: InteractiveLive/Android/VideoOne/solutions/interactivelive/src/main/java/com/vertcdemo/solution/interactivelive/core/annotation/MessageType.java
// Copyright (c) 2023 BytePlus Pte. Ltd.
// SPDX-License-Identifier: Apache-2.0

package com.vertcdemo.solution.interactivelive.core.annotation;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({
        MessageType.MSG,
        MessageType.GIFT,
        MessageType.LIKE,
})
@Retention(RetentionPolicy.SOURCE)
public @interface MessageType {
    int MSG = 1;
    int GIFT = 2;
    int LIKE = 3;
}

File: Client/Android/solutions/vod-live-feed/src/main/java/com/byteplus/vodlive/annotation/MessageType.kt
// Copyright (c) 2023 BytePlus Pte. Ltd.
// SPDX-License-Identifier: Apache-2.0

package com.byteplus.vodlive.annotation

import androidx.annotation.IntDef


@IntDef(
    MessageType.MSG, MessageType.GIFT, MessageType.LIKE
)
@Retention(AnnotationRetention.SOURCE)
annotation class MessageType {
    companion object {
        const val MSG: Int = 1
        const val GIFT: Int = 2
        const val LIKE: Int = 3
    }
}
